src/algorithm/hnswlib/visited_list_pool.h

```diff
@@ -109,14 +105,17 @@ class VisitedListPool {
     }
 
     ~VisitedListPool() {
-        while (pool.size()) {
+        while (!pool.empty()) {
```

Comment:
use `not` instead of `!` to make it clear

Reply:
done